xunit-autolabeler-v2/ast_parser/python/source_parsers/direct_invocation.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Any, List
+
+
+from ast_parser.python.drift_data_dict import make_drift_data_dict
+from ast_parser.lib.constants import IGNORED_METHOD_NAMES
+
+
+def parse(nodes: List[Any], class_name: str) -> List[Any]:
```

Comment:
Any chance the AST node has a type declared somewhere?

Reply:
[Probably?](https://docs.python.org/3/library/ast.html) `ast.AST` is the generic type, but I'll try `FunctionDef` here too.

Reply:
Apparently `mypy` is not a fan of superclasses 🙁

(I tried typing these as the parent `ast.AST` class, but it complained about descendant class' attributes not existing on the parent class. I also tried `TypeVar('ASTNode', bound=ast.AST)` to no avail.)

Reply:
Per our chat offline - using `Any` here to please `mypy`.